cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/transform/JavaScriptHelpers.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.etl.transform;
+
+import java.util.Map;
+
+/**
+ * Utility methods defined in the JavaScript of {@link ScriptTransform}.
+ */
+public interface JavaScriptHelpers {
+  /**
+   * Converts a Java {@link Map} into a JavaScript array.
+   *
+   * @param map the Java {@link Map}
+   * @return the JavaScript array
+   */
+  Object map2JSObject(Map<? extends Object, ? extends Object> map);
```

Comment:
About my response to the previous comment, I was thinking it makes sense to treat this as a Java interface, but keep in mind that it's implemented in JS. So I thought it made sense to make room for support for `StructuredRecord` to JS object conversion, so I named the method `map2JSObject` instead of `toJSObject`.